README.md

```diff
@@ -186,6 +184,14 @@ this:
   Artsy staff should find follow the instructions in
   https://github.com/artsy/potential/blob/master/platform/Kubernetes.md#hokusai
 
+### Development
```

Comment:
I think this needs to be indented further, so it's in the "Docker and Kubernetes setup" section. Otherwise it reads as if we're recommending doing development on docker, and I don't think we're there yet. 😅